modules/editor/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@nebula.gl/editor",
   "description": "A suite of 3D-enabled data editing layers, suitable for deck.gl",
   "license": "MIT",
-  "version": "0.17.3",
+  "version": "0.17.4-alpha.2",
```

Comment:
We din't use -alpha in the past.

Reply:
We just did that so we could publish before merging to master. We'll make sure it is not alpha when we publish next.